UnitystationLauncher/Views/MainWindow.xaml.cs

```diff
@@ -1,19 +1,112 @@
 using Avalonia;
 using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+using Avalonia.Controls.Shapes;
+using Avalonia.Input;
 using Avalonia.Markup.Xaml;
+using System.Runtime.InteropServices;
 
 namespace UnitystationLauncher.Views
 {
     public class MainWindow : Window
     {
+
+        private Button _minimiseButton; //CustomButton Window
+        private Button _closeButton; //CustomButton Window
+        private Button _maximizeeButton; //CustomButton Window
+        private Path _maximizeIcon; //Update Icon
+        private ToolTip _maximizeToolTip; //Update ToolTip 
+        private DockPanel _titleBar; //For move WINDOW and hiding if user don't use Windows.
+        private Border _contentcontrol;  //If user don't use Windows OS then Grid.Row update and Grid.RowSpan
```

Comment:
we don't use _ before private variables in this code base.
Also, if you add comments to variables do them as XML doc:
```
/// <summary>
/// For move WINDOW and hiding if user don't use Windows.
/// </summary>
```
Also, improved grammar would be nice, I think this means to say:
```cs
private DockPanel _titleBar; // This is used for moving the window when the titlebar is grabbed, also for disabling on non-windows OSs
```